transport/transport.go

```diff
@@ -63,13 +63,11 @@ type recvMsg struct {
 	err error
 }
 
-func (recvMsg) isItem() bool {
-	return true
-}
+func (*recvMsg) isItem() {}
 
 // All items in an out of a recvBuffer should be the same type.
 type item interface {
-	isItem() bool
```

Comment:
I don't know. I didn't write this code originally. I'm just saying that this could be a reason why the `bool` is here.

You didn't mention this change in the commit message. Making unnecessary or unrelated changes at the same time as a real change makes it harder to review. Please try to avoid that.